### DISCUSSION TITLE
How to unselect an element type?

### DISCUSSION THREAD

**Discussion by giiyms** [pyansys/pymapdl-reader#339] [Q&A] [answered]
Hello,

Is it possible to unselect some elements and only plot the result/grid for that?

I want to unselect FLUID116 elements and plot the temperatures after the fact.
I see it in the _element_map, I tried setting the matching indexes from that map to np.nan but no success.

Is there an easier way?

**Comment by mikerife** [comment, marked as answer]
Hi @giiyms 
If you want to stick with PyMAPDL Reader you can use the element_components field to define the set of elements to plot.  The component would have needed to be defined prior to solving.

Alternatively you could use PyDPF which has element & node scoping (selecting).  Or use PyMAPDL.

Can you share a representative test model?

Mike
